docs/config.json

```
@@ -596,6 +596,11 @@
               "slug": "/access-controls/access-request-plugins/ssh-approval-discord/",
               "forScopes": ["enterprise", "cloud"]
             }
+            {
+              "title": "Access Monitoring Rules",
+              "slug": "/access-controls/access-request-plugins/access-monitoring-rules/",
```

I would actually rename this to notification routing rules since it makes more sense for access plugins.
```suggestion
              "title": "Notification Routing Rules",
              "slug": "/access-controls/access-request-plugins/notification-routing-rules/",
```